internals/webpack/webpack.dev.babel.js

```diff
@@ -19,7 +19,6 @@ module.exports = require('./webpack.base.babel')({
 
   // Don't use hashes in dev mode for better performance
   output: {
-    filename: '[name].js',
```

Comment:
@vbojilova any reason for removing this? (if we have to we should probably also remove that related comment "Don't use hashes in dev mode for better performance")